src/main/java/org/jabref/logic/importer/fetcher/DoiFetcher.java

```diff
@@ -84,4 +100,23 @@ private void doPostCleanup(BibEntry entry) {
             return Collections.emptyList();
         }
     }
+
+    /**
+     * Returns registration agency. Null if no agency is found.
+     *
+     * @param doi the doi to be searched
+     * @throws JSONException
```

Comment:
Please remove empty JavaDoc `@throws` annotations.

Maybe, in this case, the JsonException can be handled locally --> log the exception and return Optional.empty?

Reply:
Handled.